FLEDGE_browser_bidding_and_auction_API.md

```diff
@@ -41,12 +41,24 @@ These response blobs are sent back to the browser. A seller’s JavaScript can [
 fetch('https://www.example-ssp.com/auction', { <b>adAuctionHeaders: true</b>, … })
 </pre>
 Note that `adAuctionHeaders` only works with HTTPS requests.
-For each response blob sent back to the browser, the seller’s server attaches a response header containing the SHA-256 hash of the response blob:
-  
+For each response blob sent back to the browser, the seller’s server attaches a response header containing the base64url encoded (RFC 4648 section 5) SHA-256 hash of the response blob:
+
 ```
-Ad-Auction-Result: ba7816bf8f01cfea414140de5dae2223b00361a396177a9cb410ff61f20015ad
+Ad-Auction-Result: ungWv48Bz-pBQUDeXa4iI7ADYaOWF3qctBD_YfIAFa0=
 ```
 
+Multiple hashes can be included in a response by either repeating the
+header or by specifying multiple hashes separated by a `,` character. So
+```
+Ad-Auction-Result: ungWv48Bz-pBQUDeXa4iI7ADYaOWF3qctBD_YfIAFa0=,9UTB-u-WshX66Xqz5DNCpEK9z-x5oCS5SXvgyeoRB1k=
```

Comment:
where is the parsing of the comma done? if in the browser process, is this safe?

Reply:
We're just splitting the string which should be safe.